src/Orchard/Tasks/Locking/Migrations/FrameworkMigrations.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Orchard.Data.Migration;
+
+namespace Orchard.Tasks.Locking.Migrations {
+    public class FrameworkMigrations : DataMigrationImpl {
+
+        public int Create() {
+            SchemaBuilder.CreateTable("LockRecord", table => table
+                .Column<int>("Id", column => column.PrimaryKey().Identity())
+                .Column<string>("Name", column => column.NotNull().WithLength(256))
+                .Column<string>("Owner", column => column.WithLength(256))
+                .Column<int>("ReferenceCount")
+                .Column<DateTime>("CreatedUtc")
```

Comment:
Hmm, why is it better? It's not a count of the record itself, but rather that of references (or acquisitions) of it.